packages/components/psammead-useful-links/src/index.stories.jsx

```diff
@@ -0,0 +1,88 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withKnobs } from '@storybook/addon-knobs';
+import { inputProvider } from '@bbc/psammead-storybook-helpers';
+import { UsefulLink, UsefulLinksLi, UsefulLinksUl } from './index';
+import notes from '../README.md';
+
+const usefulCaptions = [
+  {
+    name: 'Mitocinmu da sauko da sautin labarai',
+    url: 'https://www.bbc.com/igbo/afirika-49883577',
+  },
+  {
+    name: 'Labaran BBC Hausa a text',
+    url: 'https://www.bbc.com/igbo/afirika-49872694',
+  },
+  {
+    name: 'Abokan huldar BBC Hausa',
+    url: 'https://www.bbc.com/igbo/afirika-49869003',
+  },
+  {
+    name: 'Timi Frank: Osinbajo ya maka mutum biyu',
+    url: 'https://www.bbc.com/igbo/afirika-49883189',
+  },
+  {
+    name: 'Gwaninta ba ta karbi wani dan Nijeriya',
+    url: 'https://www.bbc.com/igbo/afirika-49869001',
+  },
+];
+
+const generateStory = ({ usefulItems }) =>
+  inputProvider({
```

Comment:
You can use a `dirDecorator` here since we are not filtering the `services` that should appear on the storybook knob.

Reply:
Done now, thanks!